src/tests/EdFi.DataManagementService.Tests.E2E/Features/General/DiscoveryAPI.feature

```diff
@@ -18,7 +18,7 @@ Feature: The Discovery API provides information about the application version, s
                       "urls": {
                           "dependencies": "{BASE_URL}/metadata/dependencies",
                           "openApiMetadata": "{BASE_URL}/metadata/specifications",
-                          "oauth": "{BASE_URL}/oauth/token",
+                          "oauth": "http://localhost/oauth/token",
```

Comment:
`http://localhost` is the API response

Reply:
The backend code for the step definition is supposed to be replacing the `{BASE_URL}` token with the actual base URL for the running instance. This way we can try running the E2E tests against a different URL in the future, and this test shouldn't fail. Since the test passed, I thought i had written the step definition code correctly. See [this line](https://github.com/Ed-Fi-Alliance-OSS/Data-Management-Service/blob/fed2f612104c0a353b9c8286cdf36a0f00e2b207/src/tests/EdFi.DataManagementService.Tests.E2E/StepDefinitions/StepDefinitions.cs#L415).

Reply:
The "oauth" endpoint is the only one that's hard coded. See `ContainerSetup` class.

Reply:
OK, good point. We still need to be able to inject this value from configuration. Will need a different token other than `{BASE_URL}`.